code/game/antagonist/outsider/wizard.dm

```diff
@@ -65,7 +65,7 @@
 	wizard.current.SetName(wizard.current.real_name)
 
 /decl/special_role/wizard/equip(var/mob/living/carbon/human/wizard_mob)
-	default_outfit = pick(subtypesof(/decl/hierarchy/outfit/wizard))
+	default_outfit = pick(decls_repository.get_decls_of_subtype(/decl/hierarchy/outfit/wizard))
```

Comment:
This returns a decl rather than a path, which I don't think is supported. Either use `get_decl_paths_of_subtype()` or modify default_outfit to handle the case where it's a decl rather than a path.